File: testdata/hallucinated_deps/fake_imports.js
// Test file with hallucinated JavaScript dependencies

import fs from 'fs';  // builtin - should be ignored
import path from 'path';  // builtin - should be ignored

import express from 'express';  // real package - should pass
import lodash from 'lodash';  // real package - should pass

// These are fake packages that should be flagged
import nonexistentAiGeneratedPackage from 'nonexistent-ai-generated-package-12345';
import { helper } from 'fake-utils-library-xyz';
const madeUp = require('totally-made-up-sdk');

function main() {
    console.log("Hello, world!");
}

main();

File: testdata/hallucinated_deps/fake_imports.go
// Test file with hallucinated Go dependencies
package main

import (
	"fmt"    // stdlib - should be ignored
	"os"     // stdlib - should be ignored
	"net/http"  // stdlib - should be ignored

	"github.com/gorilla/mux"  // real package - should pass
	"github.com/spf13/cobra"  // real package - should pass

	// These are fake packages that should be flagged
	"github.com/nonexistent/ai-generated-package-12345"
	"github.com/fake/utils-library-xyz"
	"github.com/totally/made-up-sdk-abc123"
)

func main() {
	fmt.Println("Hello, world!")
}
